provision-base.sh

```diff
@@ -331,11 +366,15 @@ install_base()
 
 	if [ "$BOURNE_SHELL" = "bash" ]; then
 		install_bash
-	fi
+    elif [ "$BOURNE_SHELL" = "zsh" ]; then
+		install_zsh
+	    config_zsh_shell
```

Comment:
inconsistent use of whitespace (2 tabs - vs 1 tab + spaces). Maybe drop in a noexpandtab rule into your .vimrc file for .sh scripts?